### PR TITLE
Create initial accounts for Migrated Projects

### PR DESCRIPTION
**Pull request by DylanTinianov**
## Description

Migrated projects have no transaction executions and require initial accounts to be created upon first accessing them.

______

For contributor use:

- [ ] Targeted PR against correct branch (see [CONTRIBUTING.md](https://github.com/onflow/flow-playground/blob/master/CONTRIBUTING.md))
- [ ] Linked to Github issue with discussion and accepted design OR link to spec that describes this work.
- [ ] Code follows the [standards mentioned here](https://github.com/onflow/flow-playground/blob/master/CONTRIBUTING.md).
- [ ] Updated relevant documentation 
- [ ] Re-reviewed `Files changed` in the Github PR explorer
- [ ] Added appropriate labels

### CODE REVIEW

File: controller/projects.go [hunk 1]
```diff
@@ -151,6 +151,13 @@ func (p *Projects) Get(id uuid.UUID) (*model.Project, error) {
 		return nil, errors.Wrap(err, "failed to get project")
 	}
 
+	if proj.TransactionExecutionCount == 0 {
```
Comment (devbugging) [inline controller/projects.go:155]: Is this only relevant for migrated projects when they are first accessed? if so would it make more sense to put this inside migrations https://github.com/onflow/flow-playground-api/blob/master/migrate/migrate.go****

Comment (DylanTinianov) [reply controller/projects.go:155]: That's in master branch, doesn't exist in V2 anymore

Comment (devbugging) [reply controller/projects.go:155]: I see. OK.

Comment (devbugging) [reply controller/projects.go:155]: Just add comment explaining why is that there/

**Review by devbugging — APPROVED** [review]

**Pull request merged**